pkgs/development/python-modules/pytest-mock/default.nix

```diff
@@ -17,6 +19,8 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ setuptools_scm ];
 
+  propagatedBuildInputs = [ pytest ];
```

Comment:
pytest extensions should not propagate pytest so it is possible to use them with different versions of pytest